### PR TITLE
Install git in Docker image

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-frontend/issues/12768

### CODE REVIEW

**Comment by eviljeff** [comment]
can we have a link to whatever this fixes, or whatever broke it

File: .circleci/config.yml [hunk 1]
```diff
@@ -356,6 +364,7 @@ workflows:
   default-workflow:
     jobs:
       - build
+      - build-docker-image
```
Comment (eviljeff) [inline .circleci/config.yml:367]: we only need this for the release jobs (?) so I guess:
```suggestion
      - build-docker-image:
          filters:
            branches:
              ignore: /.*/
```

Comment (willdurand) [reply .circleci/config.yml:367]: no, that's the contrary in fact.

Comment (eviljeff) [reply .circleci/config.yml]: oh, right

**Comment by willdurand** [comment]
just realized we could also exclude the main (master) branch too

**Review by eviljeff — APPROVED** [review]

**Pull request merged**